style.css

```diff
@@ -0,0 +1,68 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Inter", sans-serif;
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+body {
+    position: relative;
+    display: grid;
+    grid-template-areas:
+        "navHead navHead navHead"
+        "nav body body"
+        "... body body"; 
+}
+
+/* nav bar */
+
+.logo{
+    color: var(--primary-color);
+}
+
+.navHead {
+    position: relative;
+    top: 0;
+    padding: 1rem;
+    z-index: 1;
+    color: var(--primary-color);
+    font-weight: bold;
+    background-color: var(--primary-bg);
+    box-shadow: 0 0 8px var(--medium-blue-gray);
+}
+
+.nav {
+    position: sticky;
+    top: 0;
+    grid-area: nav;
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+    overflow-y: scroll;
+}
+
+.nav::-webkit-scrollbar{
+    display: none;
+}
+
```

Comment:
If you wanna hide scroll bar in mozilla also add this code
/* Hide scrollbar for IE, Edge and Firefox */
.nav {
    -ms-overflow-style: none;  /* IE and Edge */
    scrollbar-width: none;  /* Firefox */
}

You can try opening you site in mozilla i think scrollbar would be there